packages/react-error-overlay/webpack.config.iframe.js

```diff
@@ -43,6 +45,24 @@ module.exports = {
       },
     ],
   },
+  optimization: {
+    minimizer: [
+      // This code is embedded as a string, so it would never be optimized
+      // elsewhere.
+      new UglifyJsPlugin({
+        uglifyOptions: {
+          compress: {
+            warnings: false,
+            comparisons: false,
+          },
+          output: {
+            comments: false,
```

Comment:
This might be a legal issue, because this removes license comments too.

Reply:
Is there an option that will allow us to preserve important comments (`/*! ... */`)?

Reply:
@iansu this setting is already used in current versions of cra. it just looks new because of shufflng. Ill dive into the uglify options but if i dont find anything we should create a seperate issue.

https://github.com/facebook/create-react-app/blob/next/packages/react-scripts/config/webpack.config.prod.js#L394
https://github.com/facebook/create-react-app/blob/next/packages/react-error-overlay/webpack.config.iframe.js#L62

Reply:
https://github.com/webpack-contrib/uglifyjs-webpack-plugin#extractcomments

Reply:
This issue is not related regarding to upgrading webpack 4, it can be done in seperate PR to keep eventual comments in minified code.